src/validation/error.messages.ts

```diff
@@ -1,6 +1,6 @@
 export enum ErrorMessages {
-  GIT_HANDLE = 'Enter the username of the GitHub handle (aka GitHub account)',
-  INVALID_GIT_HANDLE = 'Enter the username of the GitHub handle in the correct format (aka GitHub account)',
+  GIT_HANDLE = 'Enter a GitHub handle (aka GitHub username)',
+  INVALID_GIT_HANDLE = 'GitHub handles may only contain alphanumeric characters or single hyphens, and cannot begin or end with a hyphen.',
```

Comment:
Error message doesn't include max length criteria. _GitHub handles may be a maximum length of 39 characters, only contain alphanumeric characters or single hyphens, and cannot begin or end with a hyphen_